.golangci.yaml

```diff
@@ -15,70 +17,78 @@ linters:
-          - github.com/mattn/go-isatty
-          - github.com/muesli/mango-cobra
-          - github.com/muesli/roff
-          - github.com/nwidger/jsoncolor
-          - github.com/oklog/ulid/v2
-          - github.com/openfga/api
-          - github.com/openfga/cli
-          - github.com/openfga/go-sdk
-          - github.com/openfga/language
-          - github.com/openfga/openfga
-          - github.com/rung/go-safecast
-          - github.com/schollz/progressbar/v3
-          - github.com/spf13/cobra
-          - github.com/spf13/pflag
-          - github.com/spf13/viper
-          - golang.org/x/time/rate
-          - google.golang.org/protobuf/encoding/protojson
-          - google.golang.org/protobuf/types/known/structpb
-          - gopkg.in/yaml.v3
-      test:
-        files:
-          - "$test"
-        allow:
-          - $gostd
-          - github.com/golang/mock/gomock
-          - github.com/openfga/api/proto
-          - github.com/openfga/cli
-          - github.com/openfga/go-sdk
-          - github.com/openfga/openfga
-          - github.com/stretchr
-          - go.uber.org/mock/gomock
-
-  tagliatelle:
-    case:
-      use-field-name: true
+  settings:
+    depguard:
       rules:
-        json: snake
-
-  funlen:
-    lines: 120
-    statements: 80
+        main:
+          files:
+            - $all
+            - "!$test"
+          allow:
+            - $gostd
+            - github.com/gocarina/gocsv
+            - github.com/hashicorp/go-multierror
+            - github.com/mattn/go-isatty
+            - github.com/muesli/mango-cobra
+            - github.com/muesli/roff
+            - github.com/nwidger/jsoncolor
+            - github.com/oklog/ulid/v2
+            - github.com/openfga/api
+            - github.com/openfga/cli
+            - github.com/openfga/go-sdk
+            - github.com/openfga/language
+            - github.com/openfga/openfga
+            - github.com/rung/go-safecast
+            - github.com/schollz/progressbar/v3
+            - github.com/spf13/cobra
+            - github.com/spf13/pflag
+            - github.com/spf13/viper
+            - golang.org/x/time/rate
+            - google.golang.org/protobuf/encoding/protojson
+            - google.golang.org/protobuf/types/known/structpb
+            - gopkg.in/yaml.v3
+        test:
+          files:
+            - "$test"
+          allow:
+            - $gostd
+            - github.com/golang/mock/gomock
+            - github.com/openfga/api/proto
+            - github.com/openfga/cli
+            - github.com/openfga/go-sdk
+            - github.com/openfga/openfga
+            - github.com/stretchr
+            - go.uber.org/mock/gomock
 
-  goimports:
-    local-prefixes: "github.com/openfga/cli"
+    tagliatelle:
+      case:
+        use-field-name: true
+        rules:
+          json: snake
 
-issues:
-  exclude-use-default: true
-  exclude-rules:
+    funlen:
+      lines: 120
+      statements: 80
+  exclusions:
```

Comment:
I think based on the recommended changes for [`issues.exclude-rules`](https://golangci-lint.run/product/migration-guide/#issuesexclude-rules) this should be something like

```
  exclusions:
    rules:
      - linters:
          - lll
        path: cmd/tuple/write(.*).go
      - linters:
          - err113
          - funlen
          - lll
        path: _test.go
```

Reply:
Did that in [`a50f4a4`](https://github.com/openfga/cli/pull/476/commits/a50f4a447cbe6d5c500c609ab73521d146ea7e14)